lang/简体中文.json

```diff
@@ -3,19 +3,19 @@
         "terminated": "\n应用程序已终止，请关闭窗口以退出应用程序。",
         "goes_online": "{channel} 该频道已上线, 正在切换...",
         "goes_offline": "{channel} 该频道已离线, 正在切换...",
-        "claimed_drop": "声明的掉宝: {drop}",
-        "claimed_points": "声明的积分奖励: {points}",
+        "claimed_drop": "领取的掉宝: {drop}",
+        "claimed_points": "领取的积分奖励: {points}",
         "earned_points": "观看直播获得积分: {points}, 总积分: {balance}",
         "no_channel": "没有可观看的频道，等待一个在线直播频道...",
         "no_campaign": "没有可用于掉宝的活动，等待一个有效的掉宝活动..."
     },
     "login": {
-        "unexpected_content": "返回的内容类型有误通常是由于被重定向.您需要VPN才能登录吗？",
+        "unexpected_content": "返回的内容类型有误通常是由于被重定向.您需要登陆才能联网吗？",
         "incorrect_login_pass": "用户名或密码错误.",
-        "incorrect_email_code": "电子邮箱错误.",
-        "incorrect_twofa_code": "邮箱令牌错误.",
-        "email_code_required": "需要电子邮件代码 查看你的邮件.",
-        "twofa_code_required": "需要邮箱2FA令牌."
+        "incorrect_email_code": "电子邮箱验证码错误.",
+        "incorrect_twofa_code": "2FA令牌错误.",
```

Comment:
Email token（邮箱令牌） and 2FA token（2FA令牌） are the same thing, but for Chinese users, the name has always been email token（邮箱令牌）
Or modify to 邮箱2FA令牌

Reply:
Email token and 2FA token are not the same.
Verifying the email used for sign up is the basic security setting for your twitch account. Email token is used for everyone.
2FA is an extra layer of security that only use authenticator App or text messages.
![image](https://user-images.githubusercontent.com/13106792/190652860-1150c79f-a337-4abd-b0ac-17ea2792c300.png)

As you explained in the comment, Chinese users usually don't use App like Google Authenticator, nor use cell text messages. Since they only setup the Email, the only error code they will get will be "email_code_required" or  "incorrect_email_code". 
"twofa_code_required" and "incorrect_twofa_code" are error code for the users who setup 2FA, and you should tell the user to look for the 2FA code from their phone. 

Your translations for Email token(邮箱令牌) are still kept. But 2FA is just not email.

Reply:
Yes, this is two different things. Email token comes via email, 2FA token is read from a device like a mobile phone.

Unless @Suz1e would have any additional objections, I think this can stay.

Reply:
The screenshot you showed is emal code/token not 2FA. It was sent because you setup email in your account. 
The section you highlighted in red say: We strongly recommend you to setup your 2FA to protect your account. This also indicate that this email sent to you is not 2FA.

I want to translate to 手机令牌 or 双重验证码 too but you translated into 2FA令牌 at the begining. So I kept your translation and only change it from 邮箱2FA令牌 to 2FA令牌.

Again, 2FA is different than email. No need to add 邮箱 into "incorrect_twofa_code" and "twofa_code_required".